src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -115,7 +115,7 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
     };
   }
 
-  protected getFindDatesUTCAdditionalParameters(): Record<string, any> {
+  protected async getFindDatesUTCAdditionalParameters(): Promise<Record<string, any>> {
```

Comment:
Just curious why was this changed to async? fftr

Reply:
Cleanup - `BYOCLayer` already defined it as `async` (because it needs to call `await this.fetchLayerParamsFromSHServiceV3`) and it's better to be consistent. 

This meant that `findDatesUTC` code was copy-pasted to `BYOCLayer` just to add an `await`... Since there is no harm in making this function `async`, I have simply changed the default signature and cleaned up duplicated code from `BYOCLayer`.